portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -429,6 +429,11 @@
 	<td>//div[contains(@data-field-name,'${key_fieldName}')]//input[contains(@id,'inputFile')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>UPLOAD_FIELD_GUEST</td>
+	<td>//div[contains(@data-field-name,'${key_fieldName}')]//input[@type='text']</td>
```

Comment:
@brunofariasdeo, isn't the input of type `file`? Please check [here](https://github.com/liferay/liferay-portal/blob/5aceff3ec63a5118178d7967f0903964b769a773/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js#L250-L255.). Can we reflect it on this locator?

Reply:
You're correct, I have changed this locator accordingly.